fetch.bs

```diff
@@ -1869,6 +1869,12 @@ otherwise, it is unset.
 <a for=request>done flag</a>, and <a for=request>timing allow failed flag</a> are used as
 bookkeeping details by the <a for=/>fetch</a> algorithm.
 
+<p>A <a for=/>request</a> has an associated boolean
+<dfn export for=request id=concept-request-render-blocking>render-blocking</dfn>.
```

Comment:
Please remove the ID attribute here. It's not needed for new definitions.

Reply:
Done.